web-client/src/presenter/actions/StatusReportOrder/clearStatusReportOrderFormAction.ts

```diff
@@ -1,22 +1,15 @@
 import { STATUS_REPORT_ORDER_OPTIONS } from '@shared/business/entities/EntityConstants';
-import { isEmpty } from 'lodash';
 import { state } from '@web-client/presenter/app.cerebral';
 
-export const clearStatusReportOrderFormAction = ({
-  get,
-  store,
-}: ActionProps) => {
-  const documentToEdit = get(state.documentToEdit);
-  if (isEmpty(documentToEdit)) {
```

Comment:
This check was there to prevent loading an existing Status Report Order from clearing the existing data from the form. However, we no longer need this check because we have since switched to using separate routes for creating and editing a Status Report Order.